packages/generators/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@automattic/generators",
+	"version": "1.0.0",
+	"description": "Generate a sub-package under wp-calypso",
+	"main": "index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/generators"
+	},
+	"author": "Automattic Inc.",
+	"license": "GPL-2.0-or-later",
+	"bin": {
+		"generate-calypso-package": "cli.js"
+	},
+	"bugs": "https://github.com/Automattic/wp-calypso/issues",
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/HEAD/packages/generators#readme",
+	"type": "module",
+	"engines": {
+		"node": "^16.17.0",
+		"yarn": "^3.1.1"
+	},
+	"scripts": {
+		"generate": "generate-calypso-package"
+	},
+	"dependencies": {
+		"plop": "^3.1.1"
+	},
```

Comment:
Is this package intended to be published? If not, maybe we should make it `"private": true`.

Reply:
Okay, I've made it private